e2e/README.md

```diff
@@ -20,6 +20,10 @@ Ensure you have Xcode installed.
 
 Install [AppleSimulatorUtils](https://github.com/wix/AppleSimulatorUtils#installing) which is used in e2e scripts to launch the iOS simulator.
 
+### Setting up secrets for local runs
+
+An `.env` example is present at `e2e/example.env` rename this file to `.env` and populate the secrets from Google Cloud Platform. Do not commit the newly created `e2e/.env` file.
```

Comment:
I assume .env files are gitignored? Otherwise this seems like a risk, as even accidentally committing would mean the secrets are compromised.

Reply:
The `.env` file is included in `e2e/.gitignore`.